lib/events.js

```diff
@@ -337,16 +355,21 @@ function _onceWrap(target, type, listener) {
 }
 
 EventEmitter.prototype.once = function once(type, listener) {
-  if (typeof listener !== 'function')
-    throw new TypeError('"listener" argument must be a function');
+  if (typeof listener !== 'function') {
+    const errors = lazyErrors();
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'listener', 'Function');
```

Comment:
`s/Function/function/`? Most of the `ERR_INVALID_ARG_TYPE`s in the code base are using lowercased types.